libs/gost_engine/Makefile

```diff
@@ -0,0 +1,79 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gost_engine
+PKG_VERSION:=1.1.0.3
+PKG_RELEASE:=$(AUTORELEASE)
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/gost-engine/engine/archive/v$(PKG_VERSION)
+PKG_HASH:=fff725052e82c9adb5b738729b30141f61ac91fa457a4f4b5de18b8b24092f75
+
+PKG_MAINTAINER:=Artur Petrov <github@phpchain.ru>
+PKG_LICENSE:=OpenSSL
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+PKG_UNPACK:=$(HOST_TAR) -C "$(PKG_BUILD_DIR)" --strip-components=1 -xzf "$(DL_DIR)/$(PKG_SOURCE)"
+PKG_INSTALL:=
```

Comment:
Hmm?

Reply:
You can also blame me for this one.  The package does not handle `make install`, `cmake.mk` changes this to 1, so I'm undoing it here.  Then `make install` won't be called.  Again, if there's a proper way to handle this, please advise.

Reply:
set to 0.

Reply:
I don't think it will work, but, to be honest, I have not even tested it.
[include/package.mk#L319](https://github.com/openwrt/openwrt/blob/bb817bb4b8b0b546a70e45bd907ebfeea2370dcd/include/package.mk#L319):
```
Build/Install=$(if $(PKG_INSTALL),$(call Build/Install/Default,))
```
Makefile language is not really straightforward.  From the make info:

> '$(if CONDITION,THEN-PART[,ELSE-PART])'
> ...
> The first argument, CONDITION, first has all preceding and trailing whitespace stripped, then is expanded.  If it expands to any non-empty string, then the condition is considered to be true.  If it expands to an empty string, the condition is considered to be false.
> 

So you actually need an empty  (or all whitespace) string to be false.

Reply:
What to do with PKG_INSTALL:=? SET PKG_INSTALL:=0?

Reply:
If set PKG_INSTALL:=0 we have:
```
make[3]: *** No rule to make target 'install'.  Stop.
make[3]: Leaving directory '/home/user/openwrt/build_dir/target-mipsel_mips32_musl/gost_engine-1.1.0.3'
Makefile:78: recipe for target '/home/user/openwrt/build_dir/target-mipsel_mips32_musl/gost_engine-1.1.0.3/.built' failed
make[2]: *** [/home/user/openwrt/build_dir/target-mipsel_mips32_musl/gost_engine-1.1.0.3/.built] Error 2
make[2]: Leaving directory '/home/user/openwrt/feeds/packages/libs/gost_engine'
time: package/feeds/packages/gost_engine/compile#6.62#4.22#11.19
    ERROR: package/feeds/packages/gost_engine failed to build.
```